### ISSUE TITLE
Define the Giveth community covenant

### ISSUE THREAD

**Issue by WhyldWanderer** [Giveth/giveth-planning#203]
In preparation for the new Giveth Garden, we need to have our mission and vision clearly defined.

From the notion document that Gabi put forward:

"Don't have a covenant? Creating one will be a good exercise in codifying the purpose and direction of your community. You can think of a covenant as a constitution or decentralized social contract of sorts. On a practical level, it is a document, stored on IPFS, which explains what a DAO is about in plain english. It establishes the values, rules and customs community members are expected to follow."

Link to the notion document: https://www.notion.so/Gardens-Onboarding-f6b3cb888a82494c813e902604aae4a8

Remember that this will be used by Celeste to determine rulings on any future disputes.

- Work on Mission Conversation - discuss in #mission channel and update in this issue
    - What is Giveth about and What is this DAO trying to fund
    - Include our intentions and values

**Comment by laurenluz** [comment]
related to #188

**Comment by laurenluz** [comment]
Ok I wrote something that I think is a work-able 1st draft. I put in a bit about who we are, our vision, and then forked a lot from 1hive's. Was actually a little unsure of what types of proposals will be going through our Garden and if we're going to be distributing GIV to support proposals that get passed.

https://hackmd.io/fasEfiHgRKelVVxAUCya1A?both

Would appreciate your feedback @willyogo @GriffGreen @WhyldWanderer

**Comment by willyogo** [comment]
Love it! 🔥  Once the PR is up, can you please add the link to it to this spreadsheet @laurenluz? https://docs.google.com/spreadsheets/u/2/d/1CopwkQN9BwPDxTtL_PON5Bhuui9aUkC8ygtwCSrM41Y/edit?usp=sharing

**Comment by laurenluz** [comment]
@willyogo I added the PR to the spreadsheet, but I'm not sure how to add the covenant 🤔

**Comment by ForestSoleil** [comment]
We can close this issue out now, right?

**Comment by WhyldWanderer** [comment]
Although this issue is closed, I think its still important to keep in mind that this is a first draft and may change after our mvv workshops have been navigated and our goals clarified....

**Comment by laurenluz** [comment]
What's up there right now is actually just our draft MVV, but what I drafted in this issue is largely a fork of the 1hive covenant... it actually has less to do with out MVV and more to do with just basic "don't be violent or sexually harass anyone".

totally open to modifying but yeah, I don't think it will change drastically. thanks for the reminder though Ashley
